docs/en/multi_modal/llava.md

```diff
@@ -2,14 +2,13 @@
 
 LMDeploy supports the following llava series of models, which are detailed in the table below:
 
-|                Model                 | Size | Supported Inference Engine |
-| :----------------------------------: | :--: | :------------------------: |
-| llava-hf/Llava-interleave-qwen-7b-hf |  7B  |     TurboMind, PyTorch     |
-|       llava-hf/llava-1.5-7b-hf       |  7B  |     TurboMind, PyTorch     |
-|  llava-hf/llava-v1.6-mistral-7b-hf   |  7B  |          PyTorch           |
```

Comment:
why remove `llava-hf/llava-v1.6-mistral-7b-hf` and `llava-hf/llava-v1.6-vicuna-7b-hf`?